modules/sd/README.md

```diff
@@ -1,5 +1,11 @@
 # Sapphire Dragon
 
+<div><p>This purchase unlocks the Sapphire Dragon stat block for use with D&amp;D Beyond, including the description, lair actions and history of other gem dragons&nbsp;in digital format in the monster listing&nbsp;and access to the&nbsp;monster&nbsp;in the searchable listings, character builder, digital character sheet, and the encounter builder.</p><p>Proceeds from this product support&nbsp;Extra Life. You&nbsp;can head to the&nbsp;<a>D&amp;D page on Extra Life</a>&nbsp;to find out more.</p></div>
+
+<div></div>
```

Comment:
Looks like something went funny here, the raw HTML got inserted into the markdown.